db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_01_26_195401) do
+ActiveRecord::Schema[7.0].define(version: 2023_02_06_172633) do
```

Comment:
**issue:** you need to remove the changes to `db/schema.rb` from the diff, thanks!

Reply:
schema edits reversed